### ISSUE TITLE
Question about downloading dataset

### ISSUE THREAD

**Issue by lianggroup** [coperception/coperception#32]
Hi. Thanks for your excellent job. I have a question about downloading the full dataset by **Baidu Netdisk**.
Some .zip files downloaded on **Baidu Netdisk** such as `dep_front1.zip` are damaged and connot be used.
Looking forward to your answer. Thanks a lot!

**Comment by dekunma** [comment]
TL;DR: I will reupload data from the depth camera to Baidu Netdisk. In the meanwhile, you can download the undamaged copy from [Google Drive](https://drive.google.com/drive/folders/1nVmY7g_kprOX-I0Bqsiz6-zdJM-UXFXa) if you have VPN.

When uploading the dataset, I first uploaded them to Aliyun Drive from our school's machines; then I found Aliyun Drive does not allow for sharing zip files, so I downloaded the dataset to my local machine and then uploaded it to Baidu Netdisk (I could not upload to Baidu Netdisk from our school's machines directly). The files might have been damaged because of the way how Aliyun Drive handled zip files. 
I'll download the dataset from Google Drive, upload them to Baidu Netdisk, and let you know once I'm done. This process might take 1-2 days since you might know that accessing Google products in China can be a bit painful 🥲

**Comment by lianggroup** [comment]
Thank you for your response and for your efforts in uploading the dataset!

**Comment by dekunma** [comment]
@lianggroup I have just uploaded the data of depth cameras to Baidu Netdisk. 
Based on the hash values, it looks like these files were not broken last time: `dep_front[3,4,5,6].zip`, `dep_back6.zip`, but the rest were.
I'll close this issue for now, but feel free to reopen it or submit a new issue if you have any related questions.
